database/mysql/dumper.go

```diff
@@ -124,3 +126,25 @@ func (d *Dumper) bufferer(buf *bytes.Buffer, rowChan chan []*database.Cell, done
 		}
 	}
 }
+
+// Get tables either:
+// - from config (if specified) or
+// - from the db
+// but don't do both.
+func (d *Dumper) setTables() (tables []string, err error) {
+	table, err := d.config.ReadPrimaryRecord()
+	if err != nil {
+		return
+	}
+
+	if table != "" {
```

Comment:
We'll have to make sure this is very obvious in documentation

Reply:
Done